frontend/app/templates/details-new.html

```diff
@@ -45,21 +45,21 @@
                     <h2>Admission Policy</h2>
                     <div class="article-content">
                         <p>{{ school.profile.admissions_policy_type }}</p>
-                        <p><strong>Lottery Priorities</strong>:</p>
+                        <h3>Lottery Priorities</h3>:</p>
```

Comment:
The colon here for "Lottery Priorities" ends up on it's own line and looks odd:

<img width="380" alt="screenshot 2017-02-07 20 04 18" src="https://cloud.githubusercontent.com/assets/129679/22718851/b0bb3568-ed70-11e6-8834-0f6d3f892846.png">

Maybe just remove the colon?

Otherwise it looks good!